shells/dev/webpack.config.js

```diff
@@ -81,12 +75,7 @@ module.exports = {
   },
   devtool: '#cheap-eval-source-map',
   devServer: {
-    hot: true,
     quiet: true
   },
-  plugins: [
-    new FriendlyErrorsPlugin(),
-    new webpack.HotModuleReplacementPlugin(),
-    new webpack.NamedModulesPlugin()
-  ]
+  plugins: [new FriendlyErrorsPlugin()]
```

Comment:
I removed hot module replacement for the time being. I was hitting some really tough to debug issues between three webpack processes running (both shells and a microcosm example). I'm going to investigate it further.